docs-gen/assets/scss/_styles_project.scss

```diff
@@ -0,0 +1 @@
+@import 'td/code-dark'
```

Comment:
this was required for dark mode. see [docsy docs](https://www.docsy.dev/docs/adding-content/lookandfeel/#lightdark-color-themes)